lib/bitters/generator.rb

```diff
@@ -16,11 +16,12 @@ def install
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/"
       end
     end
 
     desc 'reset', 'Reset Bitters'
+    method_options path: :string
```

Comment:
This line isn't used by Bitters itself but would be called if a user runs `bitters reset --path foo`.
Before this commit, one could install Bitters to a custom path running `bitters install --path foo`, which makes `bitters reset` or `bitters remove` unusable as they weren't accepting a custom path and assuming the default one `base/`.

`remove` and `reset` methods, ends up calling [`install_path`](https://github.com/thoughtbot/bitters/pull/191/files#diff-b34568d26cd270735267e032af763741R58), which returns a path that contains the custom part if `--path` option is passed, + `/base`.

I can add some specs for the whole generator code if you think it's necessary. I'll do it then for Bourbon and Neat too (if https://github.com/thoughtbot/neat/pull/329 got merged).